woocommerce/wc-admin
# Remove Mollie promo note on install

Fixes #6452 

The note is removed from the WooCommerce.com feed in a seperate PR.

This removes any existing instances of the note on install.

### Detailed test instructions:

- If you do not currently have the Mollie note on your WooCommerce Admin home screen, you can add a test note with the correct name as follows:
    1. install the WooCommerce Admin Test Helper plugin [here](https://github.com/woocommerce/woocommerce-admin-test-helper)
    2. Go to the Admin notes tab
    3. Add an admin note with the name `wc-admin-effortless-payments-by-mollie`
    4. Go to the WCA home screen and verify that your test note is present
- The note is removed on a new version install, so either install an old version of WCA and upgrade to the current one, or trigger the install process manually:
    1. install the WCA test helper
    2. go to the Tools tab
    3. click the `Trigger WCA install` button
